src/reference/asciidoc/file.adoc

```diff
@@ -584,6 +584,18 @@ or `FileWritingMessageHandler.MessageFlushPredicate` implementation.
 The predicates are called for each open file.
 See the java docs for these interfaces for more information.
 
+[[file-timestamps]]
+==== File Timestamps
+
+By default, the destination file `lastModified` timestamp will be the time the file was created (except a rename
+in-place will retain the current timestamp).
+Starting with _version 4.3_, you can now configure `preserve-timestamp` (or `setPreserveTimestamp(true)` when using
+Java configuration).
+For `File` payloads, this will transfer the timestamp from the inbound file to the outbound (regardless of whether a
+copy was required).
+For other payloads, if the `FileHeaders.SET_MODIFIED` header (`file_setModified`) is present, it will be used to set
+the destination file's `lastModified` timestamp, as long as the header is a `Number`.
```

Comment:
OK. Makes sense, too: Postel's Low

Although we should `WARN` it for bad type, because I'm sure there will appear some user who asks an SO question 
> Why my "foo" `FileHeaders.SET_MODIFIED` isn't accepted, even when I use `preserve-timestamp = "true"`?

And angry smile.